i18n_menu/i18n_menu.admin.inc

```diff
@@ -54,10 +54,16 @@ function i18n_menu_translation_form($form, $form_state, $translation_set = NULL,
       );
     }
   }
-  $form['actions'] = array('#type' => 'actions');
-  $form['actions']['update'] = array('#type' => 'submit', '#value' => t('Save'));
+  $form['actions']['#type'] = 'actions';
```

Comment:
Confused... :wink: 

```
$form['actions'] = array('#type' => 'actions');
```
That means:  `$form['actions']['#type'] = 'actions';`

What exactly needs to get "fixed" here?